site/content/en/docs/formats/yolo_orientedbox.md

```diff
@@ -0,0 +1,157 @@
+---
+title: 'YOLO-OrientedBox'
+linkTitle: 'YOLO-OrientedBox'
+description: ''
+---
+
+## Format specification
+
+The YOLO-OrientedBox dataset format is designed for training and validating object detection models that use oriented bounding boxes (OBB). This format is especially useful for detecting objects in aerial and satellite imagery, where objects may not be aligned with the image axes. Detailed specifications for this format can be found in the official [YOLO Oriented Bounding Box documentation](https://docs.ultralytics.com/datasets/obb/).
+
+Supported annotation types:
+- `Oriented Bounding Boxes`
+
+The format supports various subset names, except classes, names, and backup.
+
+> Note, that by default, the YOLO framework does not expect any subset names, except `train` and `valid`. Datumaro supports this as an extension. If there is no subset separation in a project, the data will be saved in the `train` subset.
+
+## Import YOLO-OrientedBox dataset
+
+To create a Datumaro project with a YOLO-OrientedBox source, use the following commands:
+
+```bash
+datum create
```

Comment:
datum create did not work on my laptop, but datum project create -o my_project did